.changeset/strong-teachers-end.md

```diff
@@ -0,0 +1,6 @@
+---
+"@hashintel/block-design-system": minor
+"@hashintel/design-system": minor
```

Comment:
A few points on this:
1. minor is the third number (`major.minor.patch`), but from the diff it looks like your intent is to give a patch bump
2. changesets handles version increments, so what I'd do is set `block-design-system`'s version to `0.0.0` (as well as in its dependents), amend this to `patch`, and then changesets will bump it to `0.0.1` when we merge the Version Packages PR

Reply:
Thank you, I have addressed this in 665f5c1bc278e7acc8d696ce67f19d9711eeaf17